### ISSUE TITLE
Plugins Not Working

### ISSUE THREAD

**Issue by raffertyuy** [microsoft/chat-copilot#617]
**Describe the bug**
After enabling plugins (tried with Klarna Shopping and my previously working custom plugin), and typing in the chat, chat will show this error

```
Unable to generate bot response. Details: Error: 500: Microsoft.OpenApi.Readers.Exceptions.OpenApiUnsupportedSpecVersionException: OpenAPI specification version '' is not supported. at Microsoft.OpenApi.Readers.ParsingContext.Parse(YamlDocument yamlDocument) at Microsoft.OpenApi.Readers.OpenApiYamlDocumentReader.ReadAsync(YamlDocument input, CancellationToken cancellationToken) at Microsoft.OpenApi.Readers.OpenApiTextReaderReader.ReadAsync(TextReader input, CancellationToken cancellationToken) at Microsoft.OpenApi.Readers.OpenApiStreamReader.ReadAsync(Stream input, CancellationToken cancellationToken) at Microsoft.SemanticKernel.Functions.OpenAPI.OpenApi.OpenApiDocumentParser.ParseAsync(Stream stream, Boolean ignoreNonCompliantErrors, CancellationToken cancellationToken) at Microsoft.SemanticKernel.Functions.OpenAPI.Extensions.KernelOpenApiPluginExtensions.RegisterOpenApiPluginAsync(IKernel kernel, String pluginName, OpenApiFunctionExecutionParameters executionP...
```

<img width="778" alt="image" src="https://github.com/microsoft/chat-copilot/assets/1037626/189a563f-29c8-4365-bb25-9af65c6c36b4">

_Note: This works in a previous version that I still have deployed._

**To Reproduce**
1. Go to Plugins
2. Enable Klarna Shopping
3. Type message `I'm looking for shoes, size 10.`

**Platform**
 - OS: Windows
 - IDE: VS Code
 - Language: C#
 - Source: Latest code as of this issue creation date.

**Additional context**
Running locally, with `./start.ps1`

**Comment by mgoelz** [comment]
can confirm that bug.

**Comment by vicentegarciadiez** [comment]
I'm getting the same error enabling the websearch pluging and using the Azure infrastructure provided by copilot development team

**Comment by momuno** [comment]
thanks @raffertyuy  When you say latest code, are you referring to the latest release, or the latest commit on main?  Can you please point me to the exact commit you are using?

**Comment by raffertyuy** [comment]
> thanks @raffertyuy  When you say latest code, are you referring to the latest release, or the latest commit on main?  Can you please point me to the exact commit you are using?

Hi @momuno, it's the latest commit on the chain as of yesterday.

Should be this one: https://github.com/microsoft/chat-copilot/commit/ccf79dcd194464dc82014221845a2ea05122ccce

**Comment by momuno** [comment]
Thank you @raffertyuy.  We were able to reproduce and are working on reverting the version bump. (Tested that plugins do work on commit prior to the version bump).  We will then look into what caused the problem.

**Comment by darakawa** [comment]
It seems that SK now expects json with camel casing and CC follows OpenAi’s documentation, snake case. 
When I removed the attributes forcing snake case in websearch plugin, SK could deserialize and import, but CC breaks because it’s expecting the previous format.

**Comment by glahaye** [comment]
Thank you for bringing this issue up.

This should now be fixed with PR #624

**Comment by raffertyuy** [comment]
Confirmed working on my custom plugin. Klarna shopping isn't working still though, but different "Oops..." error.

**Comment by glahaye** [comment]
Another fix is coming for Klarna. It will be in when this PR is merged: #626
